metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/KIBANA/5.6.2/package/scripts/kibana_master.py

```diff
@@ -24,6 +24,7 @@
 
 from ambari_commons.os_check import OSCheck
 from ambari_commons.os_family_impl import OsFamilyFuncImpl, OsFamilyImpl
```

Comment:
Is the ansible module something we are going to distribute?

Reply:
Hm, I'm not sure where that came from. I suspect IntelliJ added that in the middle of my edits.

Reply:
Heh, yep. That extra import broke the Kibana install hard. I've made most of your requested changes and will have another commit coming shortly.